modules/plugins/languages/nix.nix

```diff
@@ -27,12 +27,12 @@
   servers = {
     rnix = {
       package = pkgs.rnix-lsp;
-      internalFormatter = cfg.format.type == "nixpkgs-fmt";
+      internalFormatter = cfg.format.type == "nixfmt";
       lspConfig = ''
         lspconfig.rnix.setup{
           capabilities = capabilities,
         ${
-          if (cfg.format.enable && cfg.format.type == "nixpkgs-fmt")
+          if (cfg.format.enable && cfg.format.type == "nixfmt")
```

Comment:
Did you test this? rnix is mostly abandoned, and I am not sure how well it supports nixfmt.

Reply:
Considering that it's also fully deprecated in nvf itself, I'm going to remove it.